src/R/Components/Impl/InteractiveWorkflow/Implementation/RInteractiveEvaluator.cs

```diff
@@ -78,10 +78,10 @@ public sealed class RInteractiveEvaluator : IInteractiveEvaluator {
             } catch (RHostBinaryMissingException) {
                 await _coreShell.ShowErrorMessageAsync(Resources.Error_Microsoft_R_Host_Missing);
                 return ExecutionResult.Failure;
-            } catch (RHostDisconnectedException) {
-                // We don't 
+            } catch (RHostDisconnectedException rhdex) {
                 return ExecutionResult.Success;
-            } catch (Exception) {
+            } catch (Exception ex) {
+                CurrentWindow.WriteError(Environment.NewLine + ex.Message);
```

Comment:
I noticed. I initially tried Failure but it looked weird without prompt. It was not immediately clear to click reset.